### ISSUE TITLE
Report saved with save_as_html uses non-exisiting v0.1.6 of dist.js. Panel example no longer works.

### ISSUE THREAD

**Issue by MarcSkovMadsen** [lux-org/lux#392]
I was trying to add a Lux app to my site awesome-panel.org based on https://lux-api.readthedocs.io/en/latest/source/guide/export.html?highlight=panel#exporting-to-panel.

```python
import html

import pandas as pd
import panel as pn
import lux

pn.extension(sizing_mode="stretch_width")

df = pd.read_csv("https://raw.githubusercontent.com/lux-org/lux-datasets/master/data/hpi.csv")

def save_as_iframe(df, style="width:100%;height:100%", frameborder="0"):
    html_content = df.save_as_html(output=True)
    html_content = html.escape(html_content)
    return f"""<iframe srcdoc="{html_content}" style={style} frameborder="{frameborder}"
"allowfullscreen></iframe>"""

lux_iframe_report = save_as_iframe(df)

# Can display in notebook
lux_panel = pn.pane.HTML(lux_iframe_report, height=425)

# Can display as app with panel serve
pn.template.FastListTemplate(site="💡 Lux and Panel", title="Analysis of Happy Planet Index Dataset", main=[lux_panel]).servable();
```

I can see that this example no longer works as it looks for the non-existing `https://unpkg.com/luxwidget@0.1.6/dist/index.js`. Only version 0.1.5 exists on unpkg.

![image](https://user-images.githubusercontent.com/42288570/123554400-1bf8ad80-d780-11eb-9e13-809af3d06219.png)

![image](https://user-images.githubusercontent.com/42288570/123554409-29ae3300-d780-11eb-8f6c-48147b4606e1.png)

**Comment by MarcSkovMadsen** [comment]
I'm on lux-api 0.3.1 and lux-widget 0.1.6

**Comment by MarcSkovMadsen** [comment]
## Workaround

Downgrading to lux-widget==0.1.5 solves the problem

**Comment by dorisjlee** [comment]
Hi @MarcSkovMadsen, Thanks for reporting this! I released the lux-widget 0.1.6 yesterday, but I realized that I had not updated the npm version (which the save_to_html uses). I've now released the [latest version on npm](https://www.npmjs.com/package/luxwidget) too, so this should be working now! Sorry about the confusion!
